languagetool-core/src/main/proto/result_cache.proto

```
@@ -0,0 +1,64 @@
+syntax = "proto3";
+package lt_ml_server;
+option java_package = "org.languagetool.tools.Cache";
+option java_outer_classname = "ProtoResultCache";
+import "ml_server.proto";
```

Buf (1.47.2)

import "ml_server.proto": file does not exist

(COMPILE)